src/components/Icon/icons/Lock.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+export default function Lock(props: any) {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" {...props}>
```

Comment:
this needs a `viewBox` attribute. note that Lock's height/width did not match the 320x320 most other icons had. you'll probably need to make a pass to make sure all React icons' viewBox/height/width attributes match their original SVG